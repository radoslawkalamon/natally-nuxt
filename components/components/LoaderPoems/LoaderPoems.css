@keyframes animation-loader-poems {
  0% {
    transform: rotate(0deg);
  }

  100% {
    transform: rotate(360deg);
  }
}

.loader-poems {
  animation-duration: 1s;
  animation-iteration-count: infinite;
  animation-name: animation-loader-poems;
  animation-timing-function: linear;
  background-color: var(--color-accent);
  height: calc(var(--font-size-base) / 4);
  margin: 0 auto;
  width: calc(var(--font-size-base) * 2);
}
