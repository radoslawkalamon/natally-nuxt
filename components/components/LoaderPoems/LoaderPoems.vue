<template>
  <div class="loader-poems" />
</template>

<script src="./LoaderPoems.ts" lang="ts"></script>
<style src="./LoaderPoems.css" scoped></style>
