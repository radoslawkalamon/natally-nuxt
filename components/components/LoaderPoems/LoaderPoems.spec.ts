import LoaderPoems from '@/components/components/LoaderPoems/LoaderPoems.vue'
import { shallPassUnitSanityTest } from '@/devtools/jest.common.spec'

describe('Components / Loader Poems', () => {
  describe('Unit', () => {
    shallPassUnitSanityTest({ component: LoaderPoems })
  })
})
