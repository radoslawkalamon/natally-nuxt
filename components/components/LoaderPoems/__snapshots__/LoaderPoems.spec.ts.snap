// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components / Loader Poems Unit shall pass unit sanity test 1`] = `"<div class=\\"loader-poems\\"></div>"`;
